tests/test-criticisms/test_ppc_plots.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import matplotlib
+matplotlib.use('Agg', warn=False)
```

Comment:
Got it. I think it's not needed. The aim of the test is just to show that the function successfully terminates.